elasticsearch-core/src/test/scala/com/sumologic/elasticsearch/restlastic/RestlasticSearchClientTest.scala

```diff
@@ -278,6 +278,48 @@ trait RestlasticSearchClientTest {
       }
     }
 
+    "Support scroll requests for multiple indices" in {
+      val index0 = indices.apply(0)
+      val index1 = indices.apply(1)
+
+      val docFutures = (1 to 15).map { n =>
+        Document(s"doc-$n", Map("ct" -> "ct", "id" -> n))
+      }.map { doc =>
+        restClient.index(index0, tpe, doc)
+      }
+      whenReady(Future.sequence(docFutures)) { _ =>
+        refresh()
+      }
+
+      val docFutures1 = (16 to 23).map { n =>
+        Document(s"doc-$n", Map("ct" -> "ct", "id" -> n))
+      }.map { doc =>
+        restClient.index(index1, tpe, doc)
+      }
+      whenReady(Future.sequence(docFutures1)) { _ =>
+        refresh()
+      }
+
+      val query = new QueryRoot(MatchAll, sortOpt = Some(Seq(SimpleSort("id", AscSortOrder))))
+      val fut = restClient.startScrollRequestIndices(Seq(index1, index), tpe, query, sizeOpt = Some(5))
```

Comment:
Why `index` instead of `index0`?

Reply:
Good catch, for the readability purpose it should be `index0`, however in the beginning be defined ` val index = indices.apply(0)`